style.css

```diff
@@ -75,6 +75,25 @@ header {
   transform: translateY(10px);
 }
 
+/* Transition hamburger to X when open */
+.open {
+  transform: rotate(90deg);
+}
```

Comment:
- [OPTIONAL] Your project looks great and also functions as expected. :+1: :+1: I'll suggest adding a smooth scroll to your CSS, it gives a pleasant scroll effect on the page as it navigates to the selected section of the webpage. You can achieve this by adding this line at the top of your CSS, `html {
  scroll-behavior: smooth;
}` :+1: :+1: :smile: